ansible/roles/stack-sunbird/templates/inbound.env

```diff
@@ -42,3 +42,11 @@ FUSIONAUTH_URL=http://fusionauth-service.{{namespace}}.svc.cluster.local:9011
 FUSIONAUTH_KEY={{fusionauth_service_admin_key}}
 
 CAMPAIGN_ADMIN_TOKEN={{uci_api_admin_token}}
+
+#Environment
+ENV={{env}}
+
+#Redis
+REDIS_HOST={{redis_host}}
```

Comment:
What are the values for these? Where are the default values defined? What about the redis index?

Reply:
I have updated the default values in the sheet.

https://docs.google.com/spreadsheets/d/1xoOXNtDR7fnW9CU4mj8Q2xZr2mkdqpXdAfCOjD9BfrY/edit#gid=965973714

Reply:
No, these cannot be added in private repo. We already have a ton of redis instance variables pre-defined. Please reuse them. We cannot be introducing new variables for each service.

Reply:
Example - https://github.com/project-sunbird/sunbird-devops/blob/release-4.7.0/ansible/roles/stack-sunbird/templates/sunbird_learner-service.env#L98-L99

Reply:
You also need to decide which redis instance you want to use. We have multiple of them. Example of another one - https://github.com/project-sunbird/sunbird-devops/blob/release-4.7.0/ansible/roles/stack-sunbird/templates/dial-service_application.conf#L171

Reply:
And what redis db index you are planning to write? You cannot be writing in DB index 0 as we already have other applications writing data. So you need to keep yourself isolated unless you are also writing similar data to same index.